go/libraries/doltcore/env/actions/docs.go

```diff
@@ -123,17 +125,17 @@ func getUpdatedWorkingAndStagedWithDocs(ctx context.Context, dEnv *env.DoltEnv,
 		root = staged
 	}
 
-	docs, err := dEnv.GetDocsWithNewerTextFromRoot(ctx, root, docDetails)
+	docs, err := env.GetDocsWithTextFromRoot(ctx, root, docDetails)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	currRoot, err = dEnv.GetUpdatedRootWithDocs(ctx, working, docs)
+	currRoot, err = env.UpdateRootWithDocs(ctx, dbData, working, env.Working, docs)
 	if err != nil {
 		return nil, nil, err
 	}
 
-	stgRoot, err = dEnv.GetUpdatedRootWithDocs(ctx, staged, docs)
+	stgRoot, err = env.UpdateRootWithDocs(ctx, dbData, staged, env.Staged, docs)
```

Comment:
This should be a method in the doltdocs package, takes a root and some docs, returns a root

Reply:
Man this causing me an import loop